skrf/network.py

```diff
@@ -845,7 +845,6 @@ def __getitem__(self, key: str | int | slice | Sized) -> Network:
         >>> d = b.s12
         """
 
-        a = self.z0  # HACK: to force getter for z0 to re-shape it (z0 getter has side effects...)
```

Comment:
keep it?

Reply:
we removed the side effects in z0's getter, so it's not needed anymore.